vendor/k8s.io/kubernetes/openshift-kube-apiserver/openshiftkubeapiserver/patch.go

```diff
@@ -152,11 +135,6 @@ func newInformers(loopbackClientConfig *rest.Config) (*kubeAPIServerInformers, e
 		OpenshiftUserInformers:     userinformer.NewSharedInformerFactory(userClient, defaultInformerResyncPeriod),
 		OpenshiftConfigInformers:   configv1informer.NewSharedInformerFactory(configClient, defaultInformerResyncPeriod),
 	}
-	if err := ret.OpenshiftUserInformers.User().V1().Groups().Informer().AddIndexers(cache.Indexers{
-		usercache.ByUserIndexName: usercache.ByUserIndexKeys,
-	}); err != nil {
-		return nil, err
-	}
 
```

Comment:
https://github.com/openshift/microshift/pull/1421/files#diff-ebfa6ec837b26117893888b31b6d9683c24f3d74583db8bc27ef515a4b754c90L184-L189

I believe, we can't start those anymore
```
i.OpenshiftQuotaInformers.Start(stopCh)
i.OpenshiftUserInformers.Start(stopCh)
```

should we try removing these two fields from the struct `kubeAPIServerInformers` since we don't support either in MicroShift
https://github.com/openshift/microshift/pull/1421/files#diff-ebfa6ec837b26117893888b31b6d9683c24f3d74583db8bc27ef515a4b754c90R142-R147

Reply:
Good idea, I can look at that.

Reply:
I've pushed updates that remove those informers and restructured the PR to include the patch file and changes to the vendor file in each commit.

@tkashem , please take another look.